test/integration/targets/collections_runtime_pythonpath/runme.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -mvEeuxo pipefail
+
+
+export PIP_DISABLE_PIP_VERSION_CHECK=1
+
+
+function _cleanup() {
+    echo Cleanup: uninstalling ansible-collections.python.dist...
+    python -m pip.__main__ uninstall -y ansible-collections.python.dist
+}
+trap _cleanup INT TERM EXIT QUIT
+
+
+>&2 echo Init: creating a virtualenv...
```

Comment:
No need to echo a message here, the output from `-x` already tells us what's going on.